edubots_surveys.Rnw

```diff
@@ -856,7 +858,6 @@ Total &  & 193 & 89 & 282 \\ \hline
 \end{table}
 
 % Olia - TODO: check for repetitions in the following paragraph that you added. thank you.
-The descriptive analysis in this case is done using a Chi-square test. For the different variables considered, some statistical significance was found for Gender, Sector, and academic Discipline. The values found with the test are: $p-value=0.0034$ for Gender, $p-value=0.0199$ for Sector, and $p-value=0.0211$ for Discipline.
```

Comment:
@oliatsi did you delete this because it was repeated?

Reply:
Yes, exactly.